apps/rush-lib/src/cli/actions/VersionAction.ts

```diff
@@ -158,8 +158,15 @@ export default class VersionAction extends BaseRushAction {
     }
 
     // Commit the package.json and change files updates.
-    git.addChanges();
-    git.commit();
+    const packageJsonUpdated: boolean = VersionControl.getUncommittedChanges().some((changePath) => {
```

Comment:
Can you reuse the result of VersionControl.getUncommittedChanges() from line 149?